library/Odesk/Phystrix/AbstractCommand.php

```diff
@@ -280,6 +279,23 @@ public function setServiceLocator(LocatorInterface $serviceLocator)
         $this->serviceLocator = $serviceLocator;
     }
 
+
+    /**
+     * Records list of actions that have taken place, with their exceptions
+     *
+     * @param string    $eventName  type from class constants EVENT_*
+     * @param Exception $e          exception that was thrown as part of $eventName
+     */
+    protected function handleEvent($eventName, \Exception $e = null )
```

Comment:
why does it accept the exception as well? we had it set and it was available already as public ->getExecutionException();